src/lib/containers/markdown/MarkdownEditor.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react'
+import IconSvg from '../IconSvg'
+import MarkdownSynapse from './MarkdownSynapse'
+
+export enum MarkdownEditorTabs {
+  WRITE = 'WRITE',
+  PREVIEW = 'PREVIEW',
+}
+
+export type MarkdownEditorProps = {
+  placeholder?: string
+}
```

Comment:
This will definitely get more complex. Will definitely need something like `onSave`. Might need a prop for "edit" use cases where you want to pre-fill the input with some value.

All of that functionality can be added as needed, though